code/modules/mining/mint.dm

```diff
@@ -102,6 +103,7 @@
 	if(T)
 		var/obj/item/O = new P(src)
 		var/obj/item/storage/bag/money/M = locate(/obj/item/storage/bag/money, T)
+		O.set_custom_materials()

```

Comment:
Don't forget to actually set the args for this proc

Reply:
dose this work